src/RadicleId.svelte

```diff
@@ -17,7 +17,7 @@
 </script>
 
 <style>
-  .urn {
+  .id {
```

Comment:
It would be nice to use a different class name than `id`, `id` makes me thing of HTML IDs in this context. Could we call it `.radicle-id` in CSS?